cmake/caches/PredefinedParams.cmake

```diff
@@ -1,3 +1,9 @@
+if (DXC_COVERAGE)
```

Comment:
Should this have the HLSL change marker comment too?

Reply:
That file is HLSL-only.

Reply:
I updated the file with a comment at the top making clear it is an HLSL-only file and describing how CMake cache scripts are used.